.coderabbit.yaml

```
@@ -0,0 +1,79 @@
+    language: "en-US"
+    path_instructions:
+      - path: "**/*.py"
+        instructions: "Generate comprehensive docstrings with parameters, return values, examples, and type hints following Google style."
+      - path: "**/*.ts"
+        instructions: "Generate JSDoc comments with parameter types, return types, and usage examples."
+  unit_tests:
+    path_instructions:
+      - path: "**/*.py"
+        instructions: "Generate comprehensive pytest tests with fixtures, mocks, edge cases, and proper assertions."
+      - path: "**/*.ts"
+        instructions: "Generate Jest/Vitest tests with proper mocking, edge cases, and TypeScript types." 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)